com.jetbrains.RustRover.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.jetbrains.RustRover</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
```

Comment:
is this the actual license? if yes then there is no point in using extra-data.

Reply:
Sorry, wrong license. Just changed the license like other jetbrains flatpak apps.